src/pages/gift/Header.js

```diff
@@ -20,55 +22,92 @@ const AccountInfoBox = ({ accountAddress }) => {
     </Media>
   );
 };
-export default function Header({ selectedAccount }) {
+export default function Header ({ selectedAccount }) {
   const history = useHistory();
+  const location = useLocation();
 
   return (
     <>
       <Navbar
-        bg="primary"
-        className="pl-5 pr-4 py-3"
+        className="px-4 py-3"
+        style={{ display: 'grid', gridTemplateColumns: '60px 1fr 60px' }}
         variant="dark"
-        expand="sm">
-        <Container>
-          <Navbar.Brand onClick={() => history.push('/')}>
-            <strong>Polkadot Gifts</strong>
-          </Navbar.Brand>
-          <Navbar.Toggle aria-controls="basic-navbar-nav" />
-          <Navbar.Collapse
-            id="basic-navbar-nav"
-            className="justify-content-end">
-            <Nav>
-              <Nav.Item>
-                <Nav.Link onClick={() => history.push('/about')}>
-                  About
-                </Nav.Link>
-              </Nav.Item>
-              <Nav.Item>
-                <Nav.Link onClick={() => history.push('/claim')}>
-                  Claim Gift
-                </Nav.Link>
-              </Nav.Item>
-              <Nav.Item>
-                <Nav.Link onClick={() => history.push('/generate')}>
-                  New Gift
-                </Nav.Link>
-              </Nav.Item>
-            </Nav>
-          </Navbar.Collapse>
+        /* expand="sm" */
+      >
+        <Navbar.Brand>
+          <a
+            href='https://polkadot.network'
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+          <img
+            width={40}
+            className='p-1 shadow-sm rounded-circle'
+            src={PolkadotCircle}
+            alt={'Polkadot'}
+          />
+          </a>
+        </Navbar.Brand>
+        <Navbar.Toggle aria-controls="basic-navbar-nav" />
+        <Navbar.Collapse
+          id="basic-navbar-nav"
+          className="justify-content-center">
+          <Nav className='nav-pills shadow-sm p-1'>
```

Comment:
should we add primary color to make the pills follow the polkadot color theme to makke them stand out? right know they are kind of light gray.

Reply:
If we go with a colourful background visual or the animated gradient, the pink button might clash with the background I think
<img width="906" alt="Screenshot 2021-06-03 at 09 34 34" src="https://user-images.githubusercontent.com/7072141/120609315-59546e80-c452-11eb-8503-c606a0bbc216.png">